product/SITC/IntlAtlas/in/Services_Hierarchy.tsv

```diff
@@ -0,0 +1,17 @@
+code	name	level	name_en	name_es	name_short_en	name_short_es	parent_id
```

Comment:
Do we want to implement the short services codes before we merge this into master?